tensorboard/webapp/header/plugin_selector_component.ng.html

```diff
@@ -15,11 +15,12 @@
 limitations under the License.
 -->
 <mat-tab-group
+  mat-stretch-tabs="false"
   class="active-plugin-list"
   [selectedIndex]="getActivePluginIndex()"
   animationDuration="100ms"
 >
-  <mat-tab *ngFor="let plugin of activePlugins">
+  <mat-tab *ngFor="let plugin of activePlugins" [disabled]="!plugin.enabled">
```

Comment:
We had some weird bugs around this disabled property before. However, this seems to all be working great.

Reply:
I just kept the code that was originally there before we made the branch.